01-intro/04-flow-control/06-code-challenges/contar-vocales/solution/contarVocales.js

```diff
@@ -0,0 +1,16 @@
+const contar = (str) => {
```

Comment:
Esta solución está un poco verbose... 🤔 ... y la línea del `if` está muy larga...

Quizás una implementación así:

```js
const countVowels = (str) => {
  const lowercased = str.toLowerCase();
  let counter = 0;

  for (let i = 0; i < lowercased.length; i++) {
    if ('aeiou'.indexOf(lowercased[i]) > -1) {
      counter += 1;
    }
  }

  return counter;
};
```

O si no queremos abstraer control de flujo con `indexOf` podría hacerse con un `for` anidado...

```js
const countVowels = (str) => {
  const vowels = 'aeiou';
  const lowercased = str.toLowerCase();
  let counter = 0;

  for (let i = 0; i < lowercased.length; i++) {
    for (let j = 0; j < vowels.length; j++) {
      if (lowercased[i] === vowels[j]) {
        counter += 1;
      }
    }
  }

  return counter;
};
```

Qué opinas?